### ISSUE TITLE
Type in unimported namespace should not be preferred over type in imported namespace

### ISSUE THREAD

**Issue by jnm2** [dotnet/roslyn#42633]
**Version Used**: VS 16.6-p1

Actual behavior:

![image](https://user-images.githubusercontent.com/8040367/77204214-a441ca80-6ac8-11ea-9df0-a1a5c66a9d59.png)

Expected behavior:

The overload for the type in the imported namespace should be shown as overload 1 and the type in the imported namespace should be listed in IntelliSense above the time in the unimported namespace.

**Comment by CyrusNajmabadi** [comment]
@genlu you fixed this right?

**Comment by genlu** [comment]
@CyrusNajmabadi No, this is a different issue, the selected item in the screenshot isn't provided by import completion.

**Comment by CyrusNajmabadi** [comment]
I can't repro this.
